src/docs/patterns/appbar.mdx

```diff
@@ -35,3 +37,37 @@ In some situations, you may want to present more information in an app bar, such
 <DemoCard repository={'search-bar'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Fapp-bar%2Fsearch-bar%2Fsearch-bar.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/search-bar/index.tsx&initialpath=search'} ionic reactNative />
 
 A very common pattern seen in many types of applications is the ability to search for data on a page. This is achieved by placing a search icon/button in the app bar. When this button is clicked, a white Search Bar slides in from the right to cover the default toolbar. As the user enters text into the search field, the data is filtered in real-time.
+
+<Divider />
+
+# Design Specifications
+
+For additional instructions, refer to [Material Design's design specs on "App bars: top"](https://material.io/components/app-bars-top#specs).
+
+## Mobile
+
+<ImageGrid images={[Specs1Image]} regularWidth />
+
+## Desktop and Tablet
+
+If certain dimensions are not specified, refer to the dimensions suggested in the mobile section above.
+
+<ImageGrid images={[Specs2Image]} regularWidth />
```

Comment:
When it's implemented in code, that icon is still going to be on a 24px background:
![image](https://user-images.githubusercontent.com/29152776/89537692-ada5a800-d7c7-11ea-8e9c-228dce1f3a29.png)

Specifying it as 8 pixels might be confusing.

Reply:
Interesting... in all the design files it is labeled 8px wide. Is it impossible to achieve this on the dev side?

Reply:
Not without a lot of hacking. The SVG is defined in a 24x24 viewbox, so all of the Paths are relative to that. You would have to actually go in and modify the paths somehow. This is on the React side...i'm not 100% sure about angular, but I suspect they're using the same viewbox for their icons.

Reply:
I got you... I should probably shy away from this one in particular then. It is funny that all the design sources (our Figma files, MD icon downloads, MD design specs) are doing the 8px wide trick, yet all our dev code is using a 24*24 viewbox

![image](https://user-images.githubusercontent.com/8997218/89580434-4c032f00-d803-11ea-8751-6a6b950de6f2.png)